core/config/config.go

```diff
@@ -96,6 +96,15 @@ func GetEnvironment() string {
 	return env
 }
 
+// Path returns path to the yaml configurations
+func Path() string {
+	configPath := os.Getenv(GetEnvironmentPrefix() + configdir)
```

Comment:
We have it in few more places:(. Sending a separate PR since it could be over several files, not scope of this PR.